### ISSUE TITLE
give 7z precedence over zipfile

### ISSUE THREAD

**Issue by jbremer** [jbremer/sflock#25]
By doing so we get the extra benefits that may be found in 7z as compared to zipfile (e.g., performance) while also adding deflate64 support (see also #10).

**Comment by idiom** [comment]
Yeah, I like this approach since instead of failing, it will produce a usable (extracted) file in the case of #10 . Doing some quick tests guess() looks to select 7zip if the supported extensions are updated, but I'm not sure this can be relied on. We could add some logic that sets 7zip as the unpacker where it is installed.
